### ISSUE TITLE
Activity log lost when restored from trashbin to shared folder 

### ISSUE THREAD

**Issue by mrow4a** [owncloud/activity#1125]
- create folder nest1 and share with group, create file in that folder
- as receiver of share, delete file, and then restore that file
- activity logs is not being restored from trashbin, and thus file is hanging without activity. 

related:
https://github.com/owncloud/enterprise/issues/5222

**Comment by NannaBarz** [comment]
Are the activity from the owner or from the one who receives the share?

**Comment by mrow4a** [comment]
yes in step 3, it is receiver of share that checks for activity. It is due to change in fileid after restoring from trashbin. Original file is not affected.

**Comment by IljaN** [comment]
@santanair We think this is fixed already. Could you please check with the customer?
